simulator/mips/mips_instr.h

```diff
@@ -326,12 +326,17 @@ class FuncInstr
         uint32 lo = NO_VAL32;
 
         FuncInstr() = delete; // constructor w/o arguments for ports
-
-        explicit
+ 
+        explicit   
         FuncInstr( uint32 bytes, Addr PC = 0,
                    bool predicted_taken = false,
                    Addr predicted_target = 0);
-
+        
+        explicit    
```

Comment:
Do we need `explicit` keyword for that constructor? I believe it matters only if it has 1 parameter.